@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -54,6 +59,7 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
           wsProtocols: ['binary'],
         })
         rfb.scaleViewport = true
+        rfb.addEventListener('disconnect', this.effects._connect)
```

Comment:
https://github.com/vatesfr/xen-orchestra/pull/5728#discussion_r618530994
Maybe it's because the RFB fails to be instantiated when the canvas doesn't exist any more. Or maybe there's another error occurring? But I don't see why the listener would stop listening, and if the event is fired by the RFB, then it will try to reconnect again. So IMO, we should at least remove the listener when the component is unmounted.